lc0/src/neural/CL/OpenCLUtils.cc

```diff
@@ -0,0 +1,38 @@
+/*
+    This file is part of Leela Zero.
+    Copyright (C) 2017 Gian-Carlo Pascutto
+
+    Leela Zero is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Leela Zero is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Leela Zero.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "OpenCLUtils.h"
+
+
+
+size_t lcm(size_t a, size_t b) {
+    if (a % b == 0) {
+        return a;
+    }
+    size_t ret = a + (b - a % b);
+    return ret;
+}
+
+size_t ceilMultiple(size_t a, size_t b) {
+    if (a % b == 0) {
+        return a;
+    }
+
+    auto ret = a + (b - a % b);
+    return ret;
```

Comment:
why not just:
return a + (b - a % b)

Reply:
one does not simply ask questions about the lczero code base. for its mysteries are deep and unfathomable